src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -576,8 +576,10 @@ public static string ToUrlSlug(string value)
             //First to lower case
             value = value.ToLowerInvariant();
 
+            value = value.RemoveAccent().ToLower();
```

Comment:
That's okay I think. Actually I believe teh part about requiring the mono-locales-extra package comes from using System.Text.Encoding, so we won't fix it here anyways.